UnoCheck/Checkups/XCodeCheckup.cs

```diff
@@ -66,8 +65,27 @@ public override Task<DiagnosticResult> Examine(SharedState history)
 								new Solutions.XcodeEulaSolution())));
 					}
 
+					//before returning we want to validate that the iOS SDK is installed
+					if (!ValidateForiOSSDK(selected))
+					{
+						ReportStatus($"Xcode.app ({selected.VersionString} {selected.BuildVersion}) is installed, but missing the iOS SDK. Usually, this occurs after a recent Xcode install or update.", Status.Error);
+
+						Spectre.Console.AnsiConsole.MarkupLine("Open Xcode to complete the installation of the iOS SDK");
+
+						return Task.FromResult(new DiagnosticResult(
+							Status.Error,
+							this,
+							new Suggestion("Run `open -a Xcode`",
```

Comment:
That suggestion will not work if copy/pasted into a terminal. IOW it won't start Xcode (no path) and, if fixed, it won't show the Component window where the SDK can be loaded.

We should implement the steps from https://developer.apple.com/documentation/xcode/downloading-and-installing-additional-xcode-components#Download-Xcode-components-from-the-command-line

so we download and install the SDK from uno-check